runtime/include/gpu/rocm/chpl-gpu-gen-includes.h

```diff
@@ -86,6 +86,51 @@ __device__ static inline void chpl_gen_comm_put(void* addr, c_nodeid_t node,
   // TODO
 }
 
+__device__ __host__ static inline void chpl_gpu_write(const char *str) { printf("%s", str); }
+
+__device__ static inline void chpl_assert_on_gpu(int32_t lineno, int32_t filenameIdx) { /* no op */ }
+__host__ static inline void chpl_assert_on_gpu(int32_t lineno, int32_t filenameIdx) {
+  chpl_error("assertOnGpu() failed", lineno, filenameIdx);
+}
+
+__device__ static inline unsigned int chpl_gpu_clock(void) {
+//  return (unsigned int)clock();
+  return 0;
+}
+__host__ static inline unsigned int chpl_gpu_clock(void) {
+  return 0;
```

Comment:
I slightly prefer an internal error for these two if we are merging them like this.

Reply:
Looks like I can't call `chpl_internal_error` from within a `__device__` function as `chpl_internal_error` itself isn't a  `__device__` function.

Reply:
Ugh.. that's unfortunate. Maybe just return -1 for now as a non-sensical value rather than 0. I guess 0 is also non-sensical here, though... Hmm. I have no opinion any more. I am just a little bit scared that this might end up confusing us in a performance study.

Reply:
> Obviously we should create a device version of chpl_internal_error but that's not something I want to do for this PR.

Yeah, I think that effort is equivalent to halting from GPUs. (Let's do that in the next release. Making a note of it.)